tests/libs/datasets/timeseries_test.py

```diff
@@ -1612,3 +1612,28 @@ def test_write_read_dataset_pointer_with_source_url(tmpdir):
     assert source_url_read[CommonFields.ICU_BEDS] == [url_str1]
     # Copy to a set because the order of the URLs in the source_url may change.
     assert set(source_url_read[CommonFields.CASES]) == {url_str2, url_str3}
+
+
+@pytest.mark.parametrize("last_value,is_outlier", [(0.02, False), (0.045, True)])
```

Comment:
Add TODO about making this test stronger. The code it tests has at least 3 cases of zscore or diff being over or under a threshold.